pkg/controllers/allocation/controller.go

```diff
@@ -75,6 +78,11 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	// 1. Fetch provisioner
 	persistedProvisioner, provisionerWithDefaults, err := c.retrieveProvisionerFrom(ctx, req)
 	if err != nil {
+		if errors.IsNotFound(err) {
+			c.Batcher.Wait(&v1alpha3.Provisioner{})
+			zap.S().Errorf("No provisioner found. Create a default provisioner, or specify an alternative using the nodeSelector %s", v1alpha3.ProvisionerNameLabelKey)
```

Comment:
Tiny nit:
"Provisioner %s not found. Create the 'default' provisioner or specify an alternative using the nodeSelector %s"